### PR TITLE
 Added engineer medal

### PR DESCRIPTION
**Pull request by AsikKEsel**
## About the PR

- [x] Adding the Engineer Medal.

**Media**

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, or this PR does not require an ingame showcase

**Changelog**

:cl:
- add: added an engineer medal to the Chief Engineer locker

### CODE REVIEW

**Comment by deltanedas** [comment]
give this to atmos for setting up the ame :star:

File: Resources/Prototypes/Entities/Clothing/Neck/medals.yml [hunk 1]
```diff
@@ -19,4 +19,15 @@
...
     sprite: Clothing/Neck/Medals/gold.rsi
   - type: Clothing
-    sprite: Clothing/Neck/Medals/gold.rsi  
\ No newline at end of file
+    sprite: Clothing/Neck/Medals/gold.rsi
+
+- type: entity
+  parent: ClothingNeckBase
+  id: ClothingNeckEngineerMedal
+  name: Enginer medal
```
Comment (deltanedas) [inline Resources/Prototypes/Entities/Clothing/Neck/medals.yml:27]: enginerd1!!

use `name: engineer medal` instead

**Review by deltanedas — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/Entities/Clothing/Neck/medals.yml (hunk 1 above)
Comment (AsikKEsel) [reply Resources/Prototypes/Entities/Clothing/Neck/medals.yml]: Already changed the name

**Review by deltanedas — APPROVED** [review]

**Comment by Emisse** [comment]
I think I'd prefer just having a medal case made and given to captain

**Pull request closed without merging**